src/MacroExpander.js

```diff
@@ -202,8 +271,8 @@ export default class MacroExpander implements MacroContextInterface {
                 "need to increase maxExpand setting");
         }
         let tokens = expansion.tokens;
+        const args = this.consumeArgs(expansion.numArgs, expansion.delimiters);
```

Comment:
The reason this was a moved up is so that we consume delimiters even if there are no args, is that right?  What's an example of when this could happen?

Reply:
@kevinbarabash Any non-alphabetic characters following the control sequence is considered to be delimiters. For instance, `\def\abc|2c{123}` has delimiter `|2c`.